deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ev
+echo `zip -r extension.zip extension/`
+echo `grunt`
```

Comment:
Personally I think a separate file is cleaner. It separates the concerns, it makes it easier to read the yml file, and makes it easier to adjust later. Maybe it's only preference.

For the back ticks I don't recall what prompted me to get them in here. I'll test in my test version and then update here accordingly. If they're unnecessary I'll take them out.